src/main/java/mate/academy/spring/config/WebAppInitializer.java

```diff
@@ -10,7 +10,7 @@
 
     @Override
     protected Class<?>[] getServletConfigClasses() {
-        return new Class[]{WebConfig.class};
+        return new Class[]{WebConfig.class, SecurityConfig.class};
     }
```

Comment:
why have you added it here? and not in getRootConfigClasses()? does your app work now?

(I expect to get an answer here :)